### PR TITLE
10795 Implement login page and logout redirects

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#10795

*Description of changes:*

Commit 1:
- implemented login page
- set logout URL in misc pages
- redirect to logout URL after signout
- don't fetch org or load products if account sync timed out
- updated sbc-common-components import
- fixed a header layout issue
- fixed footer hover style
- moved static images
- misc cleanup
- app version = 1.0.10

Commit 2:
- hide component errors - _this is a workaround for a [SBC Common Component error](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/10788); what do you think?_

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: components/SbcSigninButton.vue [hunk 1]
```diff
@@ -12,22 +12,31 @@
         <v-icon class="mr-n1 pl-2">mdi-menu-down</v-icon>
       </v-btn>
     </template>
-    <!-- TODO: do we need ":fromLogin" and ":redirectOnLoginSuccess" here? -->
-    <SbcAuthMenu />
+    <SbcAuthMenu
+      :from-login="fromLogin"
+      :redirect-on-login-success="redirectOnLoginSuccess"
+    />
```
Comment (severinbeauvais) [inline components/SbcSigninButton.vue:18]: I don't think we need these props atm but now they're exposed when/if we do.

File: pages/dashboard.vue [hunk 2]
```diff
@@ -7,31 +7,33 @@
 
     <p class="dash-header-info ma-0 pt-3">Access to your BC Registries account product and services</p>
```
Comment (severinbeauvais) [inline pages/dashboard.vue:9]: Hide whitespace to see changes below.

File: pages/index.vue [hunk 3]
```diff
@@ -43,14 +43,16 @@
...
 import AccountsFooter from '~/components/AccountsFooter.vue'
 import BCOnlineAccount from '~/components/BCOnlineAccount.vue'
 import BCRegistriesAccount from '~/components/BCRegistriesAccount.vue'
 import DirectAccount from '~/components/DirectAccount.vue'
 import NoAccount from '~/components/NoAccount.vue'
 import ContactInfo from '~/components/ContactInfoBcros.vue'
+import { setLogoutUrl } from '@/utils'
 
-export default {
+export default Vue.extend ({
```
Comment (severinbeauvais) [inline pages/index.vue:55]: Needed to do this so `$config` exists on `this`.

File: pages/login.vue [hunk 4]
```diff
@@ -0,0 +1,45 @@
+<template>
```
Comment (severinbeauvais) [inline pages/login.vue:1]: This is a copy of the component in sbc-common-components, except:
- updated image source to work in Nuxt
- cleaned up layout

File: store/index.ts [hunk 5]
```diff
@@ -2,15 +2,13 @@ interface StateI {
...
   roles: string[]
-  accountId: number
 }
 
 export function state (): StateI {
   return {
     isStaff: null,
     isSbcStaff: null,
     roles: [],
-    accountId: null,
```
Comment (jordiwes) [inline store/index.ts:13]: why remove accountId?

Comment (severinbeauvais) [reply store/index.ts:13]: It was only needed in dashboard.vue so I just have it as a local variable there.

Comment (jordiwes) [reply store/index.ts:13]: Cool

**Review by jordiwes — APPROVED** [review]
LGTM

**Review by kialj876 — APPROVED** [review]
👍

**Pull request merged**